org-roam.el

```diff
@@ -629,7 +629,8 @@ Return user choice."
     (setq res
           (cond
            ((eq org-roam-completion-system 'ido)
-            (ido-completing-read prompt choices nil require-match initial-input))
+	    (let ((candidates (mapcar #'car choices)))
```

Comment:
Indentation seems a bit off.

This is a good temp fix, but we have to be sure that for all calls to `org-roam--completing-read`,  `choices` is an alist, which happens to be the case right now.